hack/jenkins/common.sh

```diff
@@ -28,7 +28,7 @@
 
 readonly OS_ARCH="${OS}-${ARCH}"
 readonly TEST_ROOT="${HOME}/minikube-integration"
-readonly TEST_HOME="${TEST_ROOT}/${OS_ARCH}-${DRIVER}-${CONTAINER_RUNTIME}-${MINIKUBE_LOCATION}-$$-${COMMIT}"
+readonly TEST_HOME="${TEST_ROOT}/$$-${COMMIT:0:7}"
```

Comment:
how about we keep the ${MINIKUBE_LOCATION}
for when debugging manually looking into the machine, if something be probalamatic, we can quickly find out what PR that is causing it

Reply:
I can but I'll have to shorten some profile names unless there's something else you think we can remove

Reply:
I replaced `COMMIT` with `MINIKUBE_LOCATION`

Reply:
minikube location alone could be problematic, if a PR get tested couple times and previous workspace as stuff in it.
can we make it like MINIKUBE_LOCATION-4Digit comm it ?